config.go

```diff
@@ -85,6 +85,10 @@ type Config struct {
 	LogLevel LogLevel
 	// LogHandler is a handler func node will send logs to.
 	LogHandler LogHandler
+
+	// tokenVerifier is a interface to allow customs tokens verifiers
+	// Zero value means that TokenVerifierJWT will be used.
+	TokenVerifier tokenVerifier
```

Comment:
I think there is no need to put this into config at moment if we do not allow to override it